app/policies/panel_policy.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class PanelPolicy < GrantPolicy
```

Comment:
in my opinion this should not inherit from GrantPolicy because there is no inheritance relationship between Grant and Panel. Grant has one Panel, but Panel is not a Grant. The only code being shared between this two policies are methods such as `user_is_grant_reviewer` and `grant_viewer_access`,  which can be extract to a separate module and be included the policy classes.

Reply:
Addressed in bbf566b